packages/calypso-e2e/src/browser-manager.ts

```diff
@@ -200,6 +200,21 @@ export async function setLoginCookie( page: Page, accountType: string ): Promise
 		throw new Error( `Cookie file ${ cookiePath } not found on disk.` );
 	}
 
+	/*
+	Check the creation time of the cookie.
+	WordPress.com cookies are valid for a limited time, typically less than 3 days.
+	So if a cookie file is found, but the creation time is older than 3 days, pretend that
+	a cookie file did not exist.
+	This is only an issue when running the tests locally.
+	*/
+	const stats = await stat( cookiePath );
+	const createdTime = Math.trunc( stats.birthtimeMs );
+	const currentTime = new Date();
+
+	if ( createdTime < currentTime.setDate( currentTime.getDate() - 3 ) ) {
+		throw new Error( `Cookie file is stale: created time is ${ stats.birthtime }` );
+	}
```

Comment:
I'm using the file creation time as a stand-in for the time when the cookie was created, but I fully acknowledge that this is not a 100% foolproof method. Anyone wanting to trick this validation can easily do so by creating a new file and copying stale cookie contents into it.

With that said, there doesn't seem to be a 'good' or 'easy' way to check the expiry timestamp of a cookie. Since we have a cookie file we're working with, it's possible to iterate through the objects, looking for the `wordpress_logged_in` name then checking the expiry time, but that is:

1. tedious, and'
2. the cookies have expiry date of +3 years but don't actually last that long.

For 2, I am basing this from my understanding of PCYsg-5YE-p2.